x12-edi/2.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,593 @@
+Connectors, Anypoint Studio, Mule flows, and Mule Global Elements.
+
+See the Release Notes for compatibility information. A link to the
+release notes is listed in the See Also section of this document.
+
+To use the X12 EDI connector in a production environment, you must
+have purchased a MuleSoft license for Anypoint B2B.
+
+NOTE: X12 EDI Connector supports files of up to 15 MB, and the memory requirement is roughly 40 to 1. For example, a 1-MB file requires up to 40 MB of memory to process, so it's important to consider this memory requirement in conjunction with your TPS needs for large X12 files. This is not an exact figure; the value might vary according to the complexity of the mapping instructions.
+
+== What's New in this Connector
+
+X12 EDI connector 2.x makes several changes to the configuration and handling implemented by X12 EDI Connector 1.x.
+
+* X12 EDI connector 1.x only supports working with transaction sets from a single version of the standard. Mule 4 X12 EDI connector 2.x supports working with transaction sets from multiple versions of the standards. The version information included in the group (GS segment) data is used to identify the standard version, and the transaction set identifier determines the particular schema.
+
+* Clearer configuration options are provided for standard X12 vs HIPAA usage. The formValidation parameter selects whether you're using standard X12, HIPAA with SNIP Type 1 validation, or HIPAA with SNIP Type 2 validation (other SNIP validation types are not implemented at the connector level, but can be implemented in your application code). Only standard X12 schemas may be used when X12 is selected; only HIPAA schemas may be used when either HIPAA option is selected.
+
+* The default character encoding is taken from the Mule platform, but can be changed to one of the standard X12 character encodings in the configuration.
+
+* Identification parameters are no longer used for the read and write operations. You can configure all parameter values in the actual message data passed to the write operation.
+
+* X12 EDI Connector 1.x provides a validateBeforeWrite configuration option
+to check for any errors in data before writing to the output stream. In the X12 EDI connector 2.x this behavior is the only mode of operation and the
+configuration parameter is no longer supported.
+
+== To Install this Connector in Anypoint Studio 7
+
+. In Anypoint Studio, click the Exchange icon in the Studio task bar.
+. Click Login in Anypoint Exchange.
+. Search for the connector and click Install.
+. Follow the prompts to install the connector.
+
+When Studio has an update, a message displays in the lower right corner, which you can click to install the update.
+
+You must change the default Anypoint configuration to use the X12 EDI connector 2.0.0.
+
+Anypoint Studio 7.1.0 and later may require more than the standard memory
+configuration to use the X12 EDI connector 2.0.0. To change the memory
```

Comment:
Kim would say: "to use X12 EDI Connector..."

Reply:
Fixed.